### PR TITLE
deposit: show form on refresh if not new

### PR DESCRIPTION
**Pull request by pedrogaudencio**
* If the form is not a fresh create instance and it was filled before, it's fully shown.

### CODE REVIEW

File: inspire/modules/deposit/static/js/deposit/literature_submission_form.js [hunk 1]
```diff
@@ -780,7 +784,31 @@ define(function(require, exports, module) {
...
+        } else if ((type === 'checkbox' || type === 'radio') && this.checked) {
+          isBlank = false;
+          return false;
+        } else if (type === 'select-multiple' && $('#subject').val() !== null) {
+          isBlank = false;
+          return false;
+        }
+      });
+
+      return isBlank;
```
Comment (jmartinm) [inline inspire/modules/deposit/static/js/deposit/literature_submission_form.js]: You can get rid of the ``isBlank`` variable as you are already returning ``false`` in the case when there is some content.

And replace this line by:

```javascript
return true;
```

Comment (pedrogaudencio) [inline inspire/modules/deposit/static/js/deposit/literature_submission_form.js]: The `return false` is just to break out of the `$.each()` so it doesn't check more fields, whereas the `isBlank` is changed accordingly and by default is set to `true`.

Comment (jmartinm) [inline inspire/modules/deposit/static/js/deposit/literature_submission_form.js]: I see, ok. You can also use ```return;``` to not confuse people like me ;)

Comment (pedrogaudencio) [inline inspire/modules/deposit/static/js/deposit/literature_submission_form.js]: The thing is that `return` something other than `false` it breaks the iteration but it continues the each cycle, only returning `false` breaks the **each**. ;)

**Comment by jmartinm** [comment]
merged 1b4f6ca

**Pull request closed without merging**